test/instrumentation/modules/http/disabling.test.js

```diff
@@ -34,7 +34,7 @@ if (cluster.isMaster) {
     }
 
     function assertSpan (t, span) {
-      t.ok(/GET localhost:\d+\//.test(span.name), 'span name')
+      t.ok(/GET localhost:\d*$/.test(span.name), 'span name')
```

Comment:
This seems odd. If allowing zero digits on the port, I'd expect the colon to be excluded as well. Should it be:

```suggestion
      t.ok(/^GET localhost:\d+$/.test(span.name), 'span name')
```

instead?

Reply:
✅ change made.